pyproject.toml

```diff
@@ -24,6 +24,7 @@ torchvision = "^0.14.1"
 torchtext = "0.14.1"
 torchdata = "0.5.1"
 torcheval = "^0.0.6"
+ipykernel = "^6.25.1"
```

Comment:
We use a little bit of cyclops in this PR, should that be added here along with into the requirements.txt?

Reply:
I think this comment still applies too?